src/yargs-interactive.js

```diff
@@ -3,8 +3,11 @@ const interactiveMode = require('./interactive-mode');
 const filterObject = require('./filter-object');
 const isEmpty = require('./is-empty');
 
+const argsProvided = process.argv.slice(2);
+const isArgProvided = (arg) => argsProvided.some(argProvided => argProvided === `--${arg}` || argProvided.startsWith(`--${arg}=`));
```

Comment:
I think _eslint_ is expecting the following:

```suggestion
const isArgProvided = (arg) => argsProvided.some((argProvided) => argProvided === `--${arg}` || argProvided.startsWith(`--${arg}=`));
```

Reply:
I forgot to check the linting, sorry about that!